Entity Explorer - Domain and URL.ipynb

```diff
@@ -9,20 +9,20 @@
     }
```

Comment:
<p>Should we collapse this into a &lt;details&gt; tag?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>